### PR TITLE
fix: the demo app running problem. Updated the dependencies

### PR DESCRIPTION
**Pull request by gnhn**
Demo app was out of date.
- Updated dependencies
- Fixed the dependency problems

### CODE REVIEW

**Comment by gnhn** [comment]
> Looks good, although it seems something is up with the config.yml?

Yes, It seems missing. Let me try to add that file

**Comment by tashmulla** [comment]
Thanks for adding the circleCI config file @gnhn 👌  I'd suggest leaving it for now and we can instead spec out [this](https://superawesomeltd.atlassian.net/browse/AAG-1980) ticket to finish the circleCI config integration? We'd created it a while back but never got round to fully speccing it out, so we can do this next sprint if you'd like?

File: app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/creatives/CreativesActivity.java [hunk 1]
```diff
@@ -53,7 +55,7 @@ protected void onCreate(Bundle savedInstanceState) {
                                 recyclerView.setItemAnimator(new DefaultItemAnimator());
                                 recyclerView.setAdapter(adapter);
 
-                                adapter.notifyDataSetChanged();
+//                                adapter.notifyDataSetChanged();
```
Comment (thomas-james-1986) [inline app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/creatives/CreativesActivity.java:58]: Do we still need this commented line?

File: app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/login/LoginActivity.java [hunk 2]
```diff
@@ -26,28 +27,29 @@ public class LoginActivity extends AppCompatActivity {
...
-                .subscribe(login::setEnabled);
-
-        RxView.clicks(login)
-                .doOnNext(aVoid -> SALoadScreen.getInstance().show(LoginActivity.this))
-                .flatMap(aVoid -> UserWorker.loginUser(LoginActivity.this, model.getUsername(), model.getPassword()).toObservable())
-                .doOnNext(logedUser -> SALoadScreen.getInstance().hide())
-                .doOnError(throwable -> SALoadScreen.getInstance().hide())
-                .subscribe(logedUser -> gotoMain(), throwable -> authError());
+//
+//        TextView username = (TextView) findViewById(R.id.UsernameTextView);
```
Comment (thomas-james-1986) [inline app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/login/LoginActivity.java:31]: Are you keeping these around for reference?

**Review by tashmulla — APPROVED** [review]
👏

**Pull request merged**